static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
fwiw, we're using swagger-codegen and haven't had issues with the optional 'q' parameter.